src/index.html

```diff
@@ -17,11 +21,401 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The stylesheet link is incorrect. It should point to a `.css` file, not a `.scss` file. SCSS files are preprocessed and should be compiled into CSS files which the browser can understand.